### ISSUE TITLE
Redis URL support

### ISSUE THREAD

**Issue by lancecarlson** [stormpath/stormpath-sdk-node#382]
createClient(); can take a url parameter. Would be nice to support it! If I submit a patch to get this working, would you take it?

**Comment by robertjd** [comment]
Hey there! It is possible to pass options down to Redis, please let me know
if this isn't documented well.

**Comment by lancecarlson** [comment]
Hmm, how can I go about it? I went around it by creating my own redis client and passing that

**Comment by robertjd** [comment]
Are you looking at these docs?
https://docs.stormpath.com/nodejs/api/client#redis

**Comment by lancecarlson** [comment]
Yes, I see a place to pass host and port, but no place to pass a URL. 

Also looking at the code itself, I noticed you were only passing parameters to the redis client using the host/port method and don't allow for passing an object (which will allow you to pass a url parameter instead)

https://github.com/stormpath/stormpath-sdk-node/blob/master/lib/cache/RedisStore.js#L22

Arity for createClient()

https://github.com/NodeRedis/node_redis#options-is-an-object-with-the-following-possible-properties
